scripts/dotnet-cli-build/TestTargets.cs

```diff
@@ -178,7 +165,7 @@ public static BuildTargetResult BuildTestAssetPackages(BuildTargetContext c)
                 {
                     "netstandard1.5",
                     "netstandard1.3",
-                    "netstandardapp1.5"
+                    "netcoreapp1.0"
```

Comment:
> It is fairly ugly and needs cleanup I agree, but it is outside the scope of this PR.

Totally agree outside of this PR. My uber point was the reason for "why" comments.